spec/jobs/scheduled/enqueue_reminders_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe Jobs::EnqueueReminders do
+  let(:user) { Fabricate(:user) }
+
+  describe '#execute' do
+    it 'Do not enqueue reminders when there are no assigned tasks' do
+      Jobs.expects(:enqueue).never
```

Comment:
We don't encourage the use of stubs in our test suite. Sidekiq provides a bunch of useful helpers that you can use. https://github.com/mperham/sidekiq/wiki/Testing